documentation/release_notes.markdown

```diff
@@ -45,6 +45,26 @@ Previous versions of Puppet Server could fail when the `config_version` setting
 
 * [SERVER-1204](https://tickets.puppetlabs.com/browse/SERVER-1204)
 
+### Known issues
+
+#### Modifications to `bootstrap.cfg` might cause problems during upgrades to 2.3.0
+
+If you modified `bootstrap.cfg` (for instance, to [enable or disable the Certificate Authority service](./configuration.html#service-bootstrapping)), upgrading to Puppet Server 2.3.0 from earlier versions of Puppet Server might fail. For instance, on Red Hat-family distributions of Linux, you might see a warning during the package update:
```

Comment:
The error listed in SERVER-1058 is specific to tk-auth. In version 2.3.0, the error will be about the versioned code service. It would look like the following:

```
2016-03-09 11:48:17,460 ERROR [main] [p.t.internal] Error during app buildup!
java.lang.RuntimeException: Service ':VersionedCodeService' not found
```

This may be worth including for easier googleability.